src/test/scala/mesosphere/marathon/integration/setup/WithMesosCluster.scala

```diff
@@ -40,7 +41,13 @@ trait WithMesosCluster extends SingleMarathonIntegrationTest { self: Suite =>
   }
 
   def startSlave(name: String, wipe: Boolean = true): Unit = {
-    ProcessKeeper.startMesos(name, s"$mesosWorkDir/$name", Seq("mesos", "slave", "--no-systemd_enable_support", s"--master=zk://${config.zkHostAndPort}/mesos", s"--hostname=$name", s"--work_dir=$mesosWorkDir/$name", s"--port=${processPort(name)}"), "registered with master", wipe)
+    val systemdFlag =
+      if (SystemUtils.IS_OS_MAC_OSX) Seq()
```

Comment:
nit: any particular reason that we didn't check for IS_OS_LINUX? https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SystemUtils.html#IS_OS_LINUX